src/collections/members/manish-tyagi/index.mdx

```diff
@@ -0,0 +1,12 @@
+---
+name: Manish Tyagi
+position: Contributor
+image_path: ./manish-tyagi.png
+github: money8203
+twitter: manishtyagi__
+linkedin: manish-tyagi-8b7b22217
```

Comment:
```suggestion
linkedin: manish-tyagi-8b7b22217
badges:
 - community
```

Reply:
I have added community badge to the @money8203 's profile , can you review this PR now @Chadha93